From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: cio: update sac values
References: bnc#445100

Symptom:     Drivers based on fcx fail to start I/O.
Problem:     Values for the sac field have changed.
Solution:    Update code accordingly.

Acked-by: John Jolly <jjolly@suse.de>
---
 arch/s390/include/asm/fcx.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: linux-sles11/arch/s390/include/asm/fcx.h
===================================================================
--- linux-sles11.orig/arch/s390/include/asm/fcx.h
+++ linux-sles11/arch/s390/include/asm/fcx.h
@@ -248,8 +248,8 @@ struct dcw {
 #define TCCB_MAX_SIZE		(sizeof(struct tccb_tcah) + \
 				 TCCB_MAX_DCW * sizeof(struct dcw) + \
 				 sizeof(struct tccb_tcat))
-#define TCCB_SAC_DEFAULT	0xf901
-#define TCCB_SAC_INTRG		0xf902
+#define TCCB_SAC_DEFAULT	0x1ffe
+#define TCCB_SAC_INTRG		0x1fff
 
 /**
  * struct tccb_tcah - Transport-Command-Area Header (TCAH)
